index.js

```diff
@@ -2,5 +2,20 @@
 'use strict';
 
 module.exports = {
-  name: 'ember-cli-font-awesome-sass'
+  name: 'ember-cli-font-awesome-sass',
+  included: function (app) {
+    this._super.included(app);
+    app.import(app.bowerDirectory + "/font-awesome/fonts/FontAwesome.otf",
+      { destDir: "assets/font-awesome/fonts" });
```

Comment:
I created issue #2 in order to consider improving destDir to be configured by an environment variable.